src/frontends/pytorch/src/op/avg_poolnd.cpp

```diff
@@ -74,4 +74,4 @@ OutputVector translate_avg_poolnd(const NodeContext& context) {
 }  // namespace op
 }  // namespace pytorch
 }  // namespace frontend
-}  // namespace ov
\ No newline at end of file
+}  // namespace ov
```

Comment:
Please remove not-required changes

Reply:
Sorry, my bad, removed. Had editor configured to automatically add newlines, probably accidently committed this change